common/list/lists-mws-en.html

```diff
@@ -0,0 +1,431 @@
+&lt;/ol&gt;</code></pre>
+	</div>
+	<div class="col-md-6">
+		<h3>Default markup - <span class="label label-success mrgn-lft-sm">Recommended</span></h3>
+		<ol>
+			<li>List item 1</li>
+			<li>
+				List item 2
+				<ol class="lst-lwr-rmn">
+					<li>List item 2a</li>
+					<li>List item 2b</li>
+				</ol>
+			</li>
+			<li>List item 3</li>
+		</ol>
+		<h4>Code</h4>
+		<pre><code>&lt;ol&gt;
+	&lt;li&gt;...
+		&lt;ol <strong>class=&quot;lst-lwr-rmn&quot;</strong>&gt;
+			&lt;li&gt;...&lt;/li&gt;
+		&lt;/ol&gt;
+	&lt;/li&gt;
+	&lt;li&gt;...&lt;/li&gt;
+&lt;/ol&gt;</code></pre>
+	</div>
+</div>
+<div class="row">
+	<h2>Roman numeral − Upper Roman</h2>
+	<div class="col-md-6">
+		<h3>Custom markup</h3>
+		<div class="lst-upr-rmn">
+			<ol>
+				<li>List item 1</li>
+				<li>
+					List item 2
+					<ol>
+						<li>List item 2a</li>
+						<li>List item 2b</li>
+					</ol>
+				</li>
+				<li>List item 3</li>
+			</ol>
+		</div>
+		<h4>Code</h4>
+		<pre><code>&lt;div <strong>class=&quot;lst-upr-rmn&quot;</strong>&gt;
+	&lt;ol&gt;
+		&lt;li&gt;...&lt;/li&gt;
+		&lt;li&gt;
+			&lt;ol&gt;
+				&lt;li&gt;...&lt;/li&gt;
+			&lt;/ol&gt;
+		&lt;/li&gt;
+		&lt;li&gt;...&lt;/li&gt;
+	&lt;/ol&gt;
+&lt;div&gt;</code></pre>
+	</div>
+	<div class="col-md-6">
+		<h3>Default markup - <span class="label label-success mrgn-lft-sm">Recommended</span></h3>
+		<ol class="lst-upr-rmn">
+			<li>List item 1</li>
+			<li>
+				List item 2
+				<ol>
+					<li>List item 2a</li>
+					<li>List item 2b</li>
+				</ol>
+			</li>
+			<li>List item 3</li>
+		</ol>
+		<h4>Code</h4>
+		<pre><code>&lt;ol <strong>class=&quot;lst-upr-rmn&quot;</strong>&gt;
+	&lt;li&gt;...&lt;/li&gt;
+	&lt;li&gt;
+		&lt;ol&gt;
+			&lt;li&gt;...&lt;/li&gt;
+		&lt;/ol&gt;
+	&lt;/li&gt;
+	&lt;li&gt;...&lt;/li&gt;
+&lt;/ol&gt;</code></pre>
+	</div>
+</div>
+<div class="row">
+	<h2>List inline</h2>
+	<div class="col-md-6">
+		<h3>Custom markup</h3>
+		<div class="list-inline">
+			<ul>
+				<li>List item 1</li>
+				<li>List item 2</li>
+				<li>List item 3</li>
+			</ul>
+		</div>
+		<h4>Code</h4>
+		<pre><code>&lt;div <strong>class=&quot;list-inline&quot;</strong>&gt;
+	&lt;ul&gt;
+		&lt;li&gt;...&lt;/li&gt;
+		&lt;li&gt;...&lt;/li&gt;
+		&lt;li&gt;...&lt;/li&gt;
+	&lt;/ul&gt;
+&lt;/div&gt;</code></pre>
+	</div>
+	<div class="col-md-6">
+		<h3>Default markup - <span class="label label-success mrgn-lft-sm">Recommended</span></h3>
+		<ul class="list-inline">
+			<li>List item 1</li>
+			<li>List item 2</li>
+			<li>List item 3</li>
+		</ul>
+		<h4>Code</h4>
+		<pre><code>&lt;ul <strong>class=&quot;list-inline&quot;</strong>&gt;
+	&lt;li&gt;...&lt;/li&gt;
+	&lt;li&gt;...&lt;/li&gt;
+	&lt;li&gt;...&lt;/li&gt;
+&lt;/ul&gt;</code></pre>
+	</div>
+</div>
+<div class="row">
+	<h2>Spacing lists - spaces added</h2>
+	<div class="col-md-6">
+		<h3>Custom markup</h3>
+		<div class="lst-spcd">
+			<ul>
+				<li>List item 1</li>
+				<li>List item 2</li>
+				<li>List item 3</li>
+			</ul>
+		</div>
```

Comment:
Add an example with `<ol>` because the CSS selector is very specific

```suggestion
		<div class="lst-spcd">
			<ul>
				<li>List item 1</li>
				<li>List item 2</li>
				<li>List item 3</li>
			</ul>
		</div>
		<div class="lst-spcd">
			<ol>
				<li>List ordered item 1</li>
				<li>List ordered item 2</li>
				<li>List ordered item 3</li>
			</ol>
		</div>
```

Reply:
Ajouté. Merci!